querybook/webapp/components/DataDocScheduleList/NextRun.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import moment from 'moment';
+import { getFutureMatches } from '@datasert/cronjs-matcher';
+import { generateFormattedDate } from 'lib/utils/datetime';
+
+export const NextRun: React.FunctionComponent<{ cron?: string }> = ({
+    cron,
+}) => {
+    if (!cron) {
+        return <span />;
+    }
+
+    const [nextDate] = getFutureMatches(cron);
```

Comment:
just checking, this would account timezones right?

Reply:
Yes, you are right.